### PR TITLE
[Feature] Spotify initial connection

### PR DESCRIPTION
**Pull request by elCarlosHez**
### What is your feature about?

It creates a context where we'll handle the Spotify connection. For now, it has a view that will show users' playlists.

### Screenshots

image

### CODE REVIEW

File: test/App.test.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ describe("App tests", () => {
       <App />,
     )
     let tree = toJson(component)
+    console.log(tree)
```
Comment (adrhem) [inline test/App.test.tsx:18]: Can you remove this console log?

File: src/models/Playlist.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+export type PlaylistQuery = {
+  href: string;
+  items: Playlist[];
+  limit: number;
+}
+
+export interface Playlist {
```
Comment (adrhem) [inline src/models/Playlist.ts]: Why is this an interface?

File: src/models/Playlist.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export type PlaylistQuery = {
```
Comment (adrhem) [inline src/models/Playlist.ts]: Can we separate both types into different files?

**Review by adrhem — APPROVED** [review]
I added non blocker comments. Otherwise looks good to me.

File: test/App.test.tsx (hunk 1 above)
Comment (elCarlosHez) [reply test/App.test.tsx:18]: yes, my bad

File: src/models/Playlist.ts (hunk 2 above)
Comment (elCarlosHez) [reply src/models/Playlist.ts]: My bad, should be a type

File: src/models/Playlist.ts (hunk 3 above)
Comment (elCarlosHez) [reply src/models/Playlist.ts]: yes, actually I consider It'd be better to rename the folder to "types"

**Pull request merged**